lib/shackles.rb

```diff
@@ -1,9 +1,11 @@
 require 'set'
 
 module Shackles
+  @@primary_environment_name = :master # Defaulting to :master for backwards compatibility. Prefer :primary.
```

Comment:
Did I do this configuration by class instance variable right? See test below for usage and additional note.

Reply:
that's a class variable, not a class instance variable. and no:

```ruby
module Shackles
  class << self
    attr_accessor :primary_environment_name
    self.primary_environment_name = :primary
  end
```

also, no need to default to :master. when someone updates to this version, it will be on them to use an older name if they prefer